README.md

```diff
@@ -304,6 +304,43 @@ If any of these steps error with, `file table overflow`, you are probably on a m
 which has a very low limit on max open files. Run `ulimit -Sn 1024` and try again.
 You'll need to do this in each new terminal you open before building Riot.
 
+Using Docker image for development
+----------------------------------
+
+Avoids installation of Node, NPM and the rest of the environment by providing
+the same environment that's used in serving Riot via Dockerfile officially
+and at the same time allows to plug in the folder with source files so that
+you can edit outside of Docker container and let it compile inside where
+it is safely separated from your machine or any custom configuration.
+
+Using this approach provides a reproducible environment between you and
+the maintainers if you share your changes in a pull request.
+
+Run it with:
+
+    # build image
```

Comment:
would rather these be stuck in a triple backtick box with language of `bash` than indented. It's just slightly more clear when reading the plaintext.